### PR TITLE
[Cherry pick] fix bugs in release 1.5

### PR DESCRIPTION
**Pull request by jerrywgz**
Fix bugs in PaddleDetection in release 1.5

### CODE REVIEW

File: PaddleCV/ssd/README.md [hunk 1]
```diff
@@ -69,14 +70,16 @@ We used RMSProp optimizer with mini-batch size 64 to train the MobileNet-SSD. Th
...
+>>>>>>> 25ff400... Support to set multiprocess in SSD model (#2776)
 ```
 
 ### Infer and Visualize
 `infer.py` is the main caller of the inferring module. Examples of usage are shown below.
-```bash
-python infer.py --dataset='pascalvoc' --nms_threshold=0.45 --model_dir='train_pascal_model/best_model' --image_path='./data/pascalvoc/VOCdevkit/VOC2007/JPEGImages/009963.jpg'
+```
+python infer.py --dataset=pascalvoc --nms_threshold=0.45 --model_dir=model/best_model --image_path=./data/pascalvoc/VOCdevkit/VOC2007/JPEGImages/009963.jpg
+>>>>>>> 25ff400... Support to set multiprocess in SSD model (#2776)
```
Comment (qingqing01) [inline PaddleCV/ssd/README.md]: need to fix conflicts.

File: PaddleCV/ssd/README.md [hunk 2]
```diff
@@ -69,14 +70,16 @@ We used RMSProp optimizer with mini-batch size 64 to train the MobileNet-SSD. Th
 You can evaluate your trained model in different metrics like 11point, integral on both PASCAL VOC and COCO dataset. Note we set the default test list to the dataset's test/val list, you can use your own test list by setting ```--test_list``` args.
 
 `eval.py` is the main caller of the evaluating module. Examples of usage are shown below.
-```bash
-python eval.py --dataset='pascalvoc' --model_dir='train_pascal_model/best_model' --data_dir='data/pascalvoc' --test_list='test.txt' --ap_version='11point' --nms_threshold=0.45
+```
+python eval.py --dataset=pascalvoc --model_dir=model/best_model --data_dir=data/pascalvoc --test_list=test.txt
+>>>>>>> 25ff400... Support to set multiprocess in SSD model (#2776)
```
Comment (qingqing01) [inline PaddleCV/ssd/README.md]: 修复冲突

File: PaddleCV/PaddleDetection/ppdet/data/data_feed.py [hunk 3]
```diff
@@ -458,12 +457,12 @@ def __init__(self,
...
-                                    std=[0.229, 0.224, 0.225],
-                                    is_scale=True,
-                                    is_channel_first=False),
-                     ResizeImage(target_size=800, max_size=1333, interp=1),
+                     DecodeImage(to_rgb=True), NormalizeImage(
+                         mean=[0.485, 0.456, 0.406],
+                         std=[0.229, 0.224, 0.225],
+                         is_scale=True,
+                         is_channel_first=False), ResizeImage(
+                             target_size=800, max_size=1333, interp=1),
```
Comment (qingqing01) [inline PaddleCV/PaddleDetection/ppdet/data/data_feed.py]: 这块还是别格式化了

File: PaddleCV/PaddleDetection/ppdet/data/data_feed.py [hunk 4]
```diff
@@ -546,17 +544,17 @@ def __init__(self,
...
+                             std=[0.229, 0.224, 0.225],
+                             is_scale=True,
+                             is_channel_first=False), ResizeImage(
+                                 target_size=800,
                                  max_size=1333,
                                  interp=1,
-                                 use_cv2=True),
-                     Permute(to_bgr=False, channel_first=True)
+                                 use_cv2=True), Permute(
+                                     to_bgr=False, channel_first=True)
```
Comment (qingqing01) [inline PaddleCV/PaddleDetection/ppdet/data/data_feed.py]: reader相关文件，单独改下， git commit --an 忽略pre-commit格式化吧

Comment (qingqing01) [reply PaddleCV/PaddleDetection/ppdet/data/data_feed.py]: Not done

File: PaddleCV/ssd/README.md (hunk 1 above)
Comment (jerrywgz) [reply PaddleCV/ssd/README.md]: Done

File: PaddleCV/ssd/README.md (hunk 2 above)
Comment (jerrywgz) [reply PaddleCV/ssd/README.md]: Done

File: PaddleCV/PaddleDetection/ppdet/data/data_feed.py (hunk 3 above)
Comment (jerrywgz) [reply PaddleCV/PaddleDetection/ppdet/data/data_feed.py]: Done

File: PaddleCV/PaddleDetection/ppdet/data/data_feed.py (hunk 4 above)
Comment (jerrywgz) [reply PaddleCV/PaddleDetection/ppdet/data/data_feed.py]: Done

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**